src/devices/sound/va_vca.h

```diff
@@ -0,0 +1,54 @@
+// license:BSD-3-Clause
+// copyright-holders:m1macrophage
+
+#ifndef MAME_SOUND_VA_VCA_H
+#define MAME_SOUND_VA_VCA_H
+
+#pragma once
+
+// Emulates a voltage-controled amplifier (VCA). The control value (CV) can
+// be set directly (set_fixed_cv()), or it can be provided in a sound stream
+// (input 1), by a device in va_eg.h, for example. The behavior of specific VCAs
+// can be emulated by using the respective configure_* functions. Note that "CV"
+// ("control value") could either refer to a control voltage, or a control
+// current, depending on the device.
```

Comment:
When the cv is provided via a stream, this is also a ring modulator.

Reply:
Actually I'm wrong, a ring modulator allows for negative values, sorry.

Reply:
...and I'm wrong when I say I'm wrong, tat's not the EQ, that's the vca.  Need more sleep.